### PR TITLE
Release chomp in Kinetic

### PR DESCRIPTION
**Pull request by 130s**
(Follow-up https://github.com/ros-planning/moveit/pull/1086#discussion_r225261670)

Hopefully this fixes the issue during bloom https://github.com/ros-planning/moveit/issues/1083#issuecomment-432622738

I also noticed that there's no ignore list for Melodic. Not yet sure why this hasn't been needed.

As far as `chomp` goes, I'm not sure if we still ignore `moveit_planners_chomp`. Looks like it's not depended from `package.xml`, so it might be ok.

```
$ git log -1 kinetic-devel
commit 2903081c904602ffeca8d71592688c93ca8adc90
Author: Isaac I.Y. Saito <130s@2000.jukuin.keio.ac.jp>
Date:   Wed Oct 24 04:23:04 2018 -0700

    0.9.13

$ ack-grep -i moveit_planners_chomp .
moveit_planners/chomp/chomp_interface/test/chomp_moveit.test
3:  <include file="$(find moveit_planners_chomp)/test/rrbot_move_group.launch"/>
5:  <test test-name="chomp_test" pkg="moveit_planners_chomp" type="chomp_moveit_test" time-limit="80.0"/>

moveit_planners/chomp/chomp_interface/test/rrbot_chomp_planning_pipeline.launch.xml
12:     <rosparam command="load" file="$(find moveit_planners_chomp)/test/chomp_planning.yaml"/>

moveit_planners/chomp/chomp_interface/test/rrbot_move_group.launch
9:  <param name="$(arg robot_description)" command="$(find xacro)/xacro --inorder '$(find moveit_planners_chomp)/test/rrbot.xacro'"/>
12:  <param name="$(arg robot_description)_semantic" textfile="$(find moveit_planners_chomp)/test/rrbot.srdf" />
16:    <rosparam command="load" file="$(find moveit_planners_chomp)/test/joint_limits.yaml"/>
35:  <include file="$(find moveit_planners_chomp)/test/rrbot_chomp_planning_pipeline.launch.xml" />
46:  <!--include file="$(find moveit_planners_chomp)/test/moveit_rviz.launch">

moveit_planners/chomp/chomp_interface/CMakeLists.txt
2:project(moveit_planners_chomp)

moveit_planners/chomp/chomp_interface/package.xml
2:  <name>moveit_planners_chomp</name>
```

### CODE REVIEW

File: indigo.ignored [hunk 1]
```diff
@@ -1,3 +1,2 @@
-chomp_motion_planner
```
Comment (rhaschke) [inline indigo.ignored]: I wouldn't change Indigo.

File: kinetic.ignored [hunk 2]
```diff
@@ -1,3 +1,2 @@
-chomp_motion_planner
 moveit_experimental
 moveit_planners_chomp
```
Comment (rhaschke) [inline kinetic.ignored]: Why not also release moveit_planners_chomp then?

**Review by rhaschke — CHANGES_REQUESTED** [review]
Don't touch Indigo. It is released, isn't it?
Fully release chomp for Kinetic and Melodic.

File: indigo.ignored (hunk 1 above)
Comment (130s) [reply indigo.ignored]: Got it. I'll fix once I get back to computer.

File: kinetic.ignored (hunk 2 above)
Comment (130s) [reply kinetic.ignored]: Sorry but I have not caught up with the development status. 
I'll look into the source to see if it's already in releaseable state.

**Comment by 130s** [comment]
@rhaschke just making sure that chomp stuff can be released into melodic as well right (the subject only contains kinetic after your change)?

File: kinetic.ignored (hunk 2 above)
Comment (130s) [reply kinetic.ignored]: I think it's fine to get ` moveit_planners_chomp` released (I posted a bit more detail at https://github.com/ros-planning/moveit/issues/1083#issuecomment-432737000).

**Comment by 130s** [comment]
Comments addressed. Please re-review.

**Review by rhaschke — APPROVED** [review]

**Pull request merged**